compass/Coordinator.java

```diff
@@ -293,7 +294,14 @@ private void start() throws InterruptedException {
         }
 
         if (!validateTransactionsToApprove(trunk, branch)) {
-          throw new RuntimeException("Trunk & branch were not consistent!!! T: " + trunk + " B: " + branch);
+          if (++validationRetries >= config.validationAttempts) {
+            throw new RuntimeException("Trunk & branch were not consistent on multiple attempts!!! T: " + trunk + " B: " + branch);
+          } else {
+            // Perform gTTA and Validation again
+            continue;
+          }
+        } else {
+          validationRetries = 0;
```

Comment:
A question do we want to call GTTA again?
why not loop over the validation?

Reply:
We need gTTA again otherwise tips will become older and older with attempts.